src/components/molecules/Comparer.svelte

```diff
@@ -0,0 +1,50 @@
+<script>
+  // Internals
+  import { reportData } from '@/stores/dataStore.js'
+
+  // Imports
+  import Counter from '../atoms/Counter.svelte'
+
+  // Retrieve reports
+  let allReports
+    reportData.subscribe((value) => {
+      allReports = value
+    })
+
+  // DD/MM/YYYY format
+  const dateOptions = {
+    day: 'numeric',
+    month: 'numeric',
+    year: 'numeric'
+  };
+
+  // set variables for todays date and last weeks date
+  const today = new Date()
+  const lastWeek = new Date()
+  lastWeek.setDate(today.getDate() - 7)
+
+  // Format dates to DD/MM/YYYY
+  const todayFormatted = today.toLocaleDateString('en-GB', dateOptions)
+  const lastWeekFormatted = lastWeek.toLocaleDateString('en-GB', dateOptions)
+
+  // Check for each date the amount of
+  // reports made that day
+  const reportsToday = allReports.filter((item) => item.date === todayFormatted).length
+  const reportsLastWeek = allReports.filter((item) => item.date === lastWeekFormatted).length
+
+  // Compare amount of reports, if the result is greater than 0, add a +
+  // Set the value of content based on if the difference is greater or smaller than 0
+  const difference = reportsLastWeek - reportsToday
+  const content = difference > 0
+    ? `+${difference}`
+    : difference
```

Comment:
Maybe you can see if you can put this functionality as a .js file in utils? Then it is not so loose in this component!